fuzzing/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use std::env;
```

Comment:
I would recommend renaming this file to indicate what exactly is being fuzzed here.

Reply:
I've renamed it